pom.xml

```diff
@@ -115,13 +115,16 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>flow-client</artifactId>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.vaadin</groupId>
             <!-- Other components are probably handy when building test views -->
             <!-- Replace artifactId with vaadin to use commercial components -->
             <artifactId>vaadin-core</artifactId>
             <scope>test</scope>
+            <!-- I using vaadin dependency in addon set scope to provided -->
```

Comment:
Should this be "If you are using vaadin..." or similar?

Reply:
Yes should be if not I